### ISSUE TITLE
Python 2.7 DeprecationWarning

### ISSUE THREAD

**Issue by Ic3fr0g** [vecxoz/vecstack#31]
```python
import vecstack
# DeprecationWarning: The module is deprecated in version 0.21 and will be removed in version 0.23 since we've dropped support for Python 2.7. Please rely on the official version of six (https://pypi.org/project/six/).
#  "(https://pypi.org/project/six/).", DeprecationWarning)
```

https://github.com/vecxoz/vecstack/blob/4909830e17469c7edbaa97ce2cde156d04906b52/vecstack/coresk.py#L57

**Comment by vecxoz** [comment]
Hi! Thanks for your contribution.

Currently vecstack is moving to the next version in which Python 2 support will be dropped and `six` deprecation issue will be fixed as well. This fix will be included in global v0.4.0 commit.

**Comment by Ic3fr0g** [comment]
Thanks! Great package btw!
